mteb/abstasks/AbsTask.py

```diff
@@ -58,3 +58,24 @@ def evaluate(self, model, split="test"):
         :param split: Which datasplit to be used.
         """
         raise NotImplementedError
+
+    @property
+    def languages(self) -> set[str]:
```

Comment:
My impression is that providing these "shortcuts" tend to be annoying for maintenance, but your mileage may vary.

Reply:
Annoying for maintenance and nice for use. My guess would be that it removes confusion due to the mapping dicts (leading to less issues). All a guess though. I will need to for:

```
MTEB(task_langs=[...]), currently uses just the user-specified splits, we should instead transfer it to use the standard language codes.
```
So rather than implement it in the MTEB object I believe it is better places here.